console.rst

```diff
@@ -38,6 +38,19 @@ For example, a command called ``CreateUser`` must follow this structure::
         }
     }
 
+If you're using the :ref:`default services.yml configuration <service-container-services-load-example>`,
```

Comment:
I like the idea! I'm not sure I did exactly what you wanted however, can you check?